CHANGELOG.rst

```diff
@@ -10,8 +10,12 @@ Known bugs:
 
 Breaking changes:
 
+- Function properties ``name`` and ``demangled_name`` in JSON are changed. Both properties are only available for ``gcc-14``. (:issue:`1059`)
```

Comment:
Should this say, `... gcc-14 or newer.`?

Reply:
There is no newer version released yet.

Reply:
I was kind of trying to make the comment forward looking as we know newer releases will have this support.

Reply:
I checked the previous changing entries and we used "... since gcc-xx".